README.md

```diff
@@ -9,7 +9,7 @@ The ue4-docker Python package contains a set of Dockerfiles and accompanying bui
 
 Key features include:
 
-- The six most recent versions of the Unreal Engine are supported (currently Unreal Engine 4.26 and newer).
+- The six most recent versions of the Unreal Engine are supported (currently Unreal Engine 4.27 and newer).
```

Comment:
This patch makes it 7 versions. Perhaps we just say "Six most recent plus 4.27-plus" (or whatever the official name for that 4.27-plus LTS branch is...) That's assuming we'll keep 4.27-plus support but drop 5.0 support when 5.6 is released.